clase-1-introduccion/ejercicios/02-tipos-de-datos.js

```diff
@@ -11,76 +11,120 @@
 
 // 👉 Paso 1: Crear una variable "nombre" de tipo string
 
+let nombre = "williams";
+
 // 👉 Paso 2: Crear una variable "edad" de tipo number
 
+let edad = 33;
+
 // 👉 Paso 3: Crear una variable "esEstudiante" de tipo boolean
 
+let esEstudiante = true;
+
 // 👉 Paso 4: Crear una variable "direccion" sin valor (undefined)
 
+let direccion;
+
 // 👉 Paso 5: Crear una variable "nulo" con valor null
 
+let nulo = null;
+
 // 👉 Paso 6: Mostrá en consola el tipo de cada variable usando typeof
 
+console.log("Tipo string", typeof nombre);    
```

Comment:
Esto deberia ser así:

```suggestion
console.log(typeof nombre, typeof edad, typeof esEstudiante, typeof direccion, typeof nulo); 
```